packages/react/hover-card/README.md

```diff
@@ -0,0 +1,13 @@
+# `react-hover-card`
+
+## Installation
+
+```sh
+$ yarn add @radix-ui/react-hover-card
+# or
+$ npm install @radix-ui/react-hover-card
+```
+
+## Usage
+
+View docs [here](https://radix-ui.com/primitives/docs/components/hover-card).
```

Comment:
Preemptively adding this as my memory isn't what it used to be.

Reply:
Haha, yep, we do the same :+1: